src/views/Auth/Auth.jsx

```diff
@@ -1,22 +1,32 @@
 import { useHistory } from 'react-router-dom';
 import { Link } from 'react-router-dom';
```

Comment:
Remember that imports from the same library can be combined into a single import statement.